packages/devtools-launchpad/src/index.js

```diff
@@ -134,20 +134,26 @@ function getTargetFromQuery() {
 }
 
 async function connectClients(actions) {
-  chrome.connectClient()
-    .then(actions.newTabs);
-
-  chrome.connectNodeClient()
-    .then(actions.newTabs);
-
   const firefoxTabs = await firefox.connectClient();
   actions.newTabs(firefoxTabs);
+
+  const chromeTabs = await chrome.connectClient();
+  actions.newTabs(chromeTabs);
+
+  const nodeTabs = await chrome.connectNodeClient();
+  actions.newTabs(nodeTabs);
```

Comment:
these should not be await because then the function waits for all three modes to finish. We should instead use `chrome.connectNodeClient().then(actions.newTabs)` to allow the function to return w/o node and chrome necessarily connecting